tests/regression/test_issue_893.cpp

```diff
@@ -0,0 +1,63 @@
+/* See https://github.com/pocl/pocl/issues/893.
```

Comment:
Please add the MIT licence and the copyright statement.

Reply:
Not fully sure about the copyright name but added it to PoCL developers.